.github/labeler.yml

```
@@ -0,0 +1,27 @@
+#
+# Pull request label rules.
+#
+# Runs in action pull-request-labeler.yml.
+#
+# See https://github.com/marketplace/actions/labeler
+#
+
+# Labels for programming languages.
+javascript: ["**/*.js", "**/*.jsx"]
+typescript: ["**/*.ts", "**/*.tsx"]
+python: "**/*.py"
+terraform: "**/*.tf"
+
+
+# Labels for components or parts of the code.
+api: api/**/*
+frontend: frontend/**/*
+analytics: analytics/**/*
+ci/cd: [".github/workflows/**/*", "bin/**/*", "**/*/Makefile"]
+docs: ["docs/**/*", "**/*.md"]
```

The folder in our repo is `documentation`. Also we have an [existing label](https://github.com/HHS/simpler-grants-gov/labels) `documentation`, should we use it?

```suggestion
documentation: ["documentation/**/*", "**/*.md"]
```